pandas/core/tools/datetimes.py

```diff
@@ -558,11 +557,16 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     4    3/12/2000
     dtype: object
 
-    >>> %timeit pd.to_datetime(s,infer_datetime_format=True)
-    100 loops, best of 3: 10.4 ms per loop
+    >>> import timeit
+    >>> timeit.timeit(
+    ...    lambda: pd.to_datetime(s, infer_datetime_format=True),
+    ...    number = 1)  # doctest: +SKIP
+    0.007785393000000029
 
-    >>> %timeit pd.to_datetime(s,infer_datetime_format=False)
-    1 loop, best of 3: 471 ms per loop
+    >>> timeit.timeit(
```

Comment:
I am not sure if it's fine to just convert to native Python compatible ones.

Reply:
I think I personally prefer to keep the `%timeit` (which is easier to use / better practice IMO). 
Another option is to add a `# doctest: +SKIP` flag for those two

Reply:
Yea .. I prefer that too but `%timeit` is specific to IPython ....

Reply:
Yes, but I think we can be opinionated about that. Maybe add a note between brackets before the example that the following only works in IPython?)

Reply:
okie!

Reply:
Actually, let me just don't add the note and revert it back with `# doctest: +SKIP`. I am sure people would know `%timeit` is for IPython ..